packages/tools/pluggable-widgets-tools/configs/tsconfig.base.json

```diff
@@ -1,39 +1,28 @@
 {
-  "compilerOptions": {
-    "noEmitOnError": true,
-    "sourceMap": true,
-    "module": "amd",
-    "target": "es5",
-    "lib": [
-      "es2015",
-      "dom"
-    ],
-    "types": [
-      "big.js",
-      "jest"
-    ],
-    "moduleResolution": "node",
-    "declaration": false,
-    "removeComments": true,
-    "noLib": false,
-    "watch": false,
-    "forceConsistentCasingInFileNames": true,
-    "noFallthroughCasesInSwitch": true,
-    "strict": true,
-    "strictFunctionTypes": false,
-    "skipLibCheck": true,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
-    "jsx": "react",
-    "jsxFactory": "createElement",
-    "allowSyntheticDefaultImports": true,
-    "esModuleInterop": true
-  },
-  "include": [
-    "../../../../src"
-  ],
-  "exclude": [
-    "../../../../dist/",
-    "../../../../node_modules/"
-  ]
+    "compilerOptions": {
+        "noEmitOnError": true,
+        "sourceMap": true,
+        "module": "es2015",
+        "target": "es5",
```

Comment:
I think for native we can override it (through [compilerOptions](https://github.com/TypeStrong/ts-loader#compileroptions) `webpack.config`) to be at least `es2015`. That should also save us quite some bundle size.

Reply:
On the same note, i don't think that we need `preset-env` for native at all. Removing it would save us also quite some bundle size